Gemfile.lock

```diff
@@ -271,4 +288,4 @@ RUBY VERSION
    ruby 2.4.0p0
 
 BUNDLED WITH
-   1.15.1
```

Comment:
Whenever you run `bundle install` on your local machine if your version of bundler is greater than the one used when generating the `Gemfile.lock` it'll update this version number. :-)